ballista/scheduler/src/bin/main.rs

```diff
@@ -117,10 +100,13 @@ async fn main() -> Result<()> {
             .init();
     }
 
-    let addr = format!("{bind_host}:{port}");
+    let addr = format!("{}:{}", opt.bind_host, opt.bind_port);
     let addr = addr.parse()?;
 
     let config = SchedulerConfig {
+        namespace: opt.namespace,
+        external_host: opt.external_host,
```

Comment:
It looks like this PR drops `bind_host`? That can be different from `external_host`, so I'm not sure we can do that.

Or maybe that is handled somewhere else in this PR and I haven't seen it yet?

Reply:
It looks like `bind_host` is not currently used in the scheduler, but it is in the executor, so this seems like an oversight.

cc @avantgardnerio who may know have opinions here

Reply:
Sorry, this is slightly confusing :). It is still used to build the bind address for the server 

```
    let addr = format!("{}:{}", opt.bind_host, opt.bind_port);
    let addr = addr.parse()?;
```

The reason `external_host` was added to `SchedulerConfig` is so that we can pass it in to `start_server` and build the `BallistaCluster` there only from the `SchedulerConfig`. In order to do that I had to pass a few more things from the `Config` struct generated by `configure_me` which isn't available in the lib code. 

So really the only point is to further minimize what we do in `main.rs` to just setting up logging and validating CLI arguments

Reply:
FWIW, I believe `external_host` is required for FlightSQL.

Reply:
I stand corrected - it appears to have morphed into its own `advertise_flight_sql_endpoint`.